client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -258,13 +258,13 @@ module.exports = {
 
 	domainManagementAddGoogleApps( pageContext ) {
 		setTitle(
-			i18n.translate( 'Add Google Apps' ),
+			i18n.translate( 'Add G Suite' ),
 			pageContext
 		);
 
 		analytics.pageView.record(
 			paths.domainManagementAddGoogleApps( ':site', pageContext.params.domain ? ':domain' : undefined ),
-			'Domain Management › Add Google Apps'
+			'Domain Management › Add G Suite'
```

Comment:
Since this is only used for analytics, we should keep it as `Google Apps` for now (especially since in other such places we've kept Google Apps, like [here](https://github.com/Automattic/wp-calypso/blob/291ce2fbe5a284562dd5e971d53fd3e48eba0b7f/client/my-sites/upgrades/controller.jsx#L144)).